services/auth-v1/constants.go

```diff
@@ -10,6 +10,7 @@ const (
 	ArgCurrent      = "current"
 	ArgCurrentShort = "C"
 	ArgContractNo   = "contract"
+	ArgNoHeaders    = "no-headers"
```

Comment:
From my point of view, we should use only one constant, from `internal/config/constants.go` since it is used across multiple services.